### PR TITLE
close #10, #30, #32, #34

### LINKED ISSUE #10: Encrypt and decrypt parts of the configuration

### PR DESCRIPTION
**Pull request by mossbeard**

### CODE REVIEW

File: src/Program.cs [hunk 1]
```diff
@@ -82,6 +83,36 @@ private static void SetLoggerSettings(string configName)
...
+            var manager = new TaskManager();
+            var certPath = @"C:\Users\MBerthold\AppData\Roaming\senseexcel\reporting\serconnector.pem";
+            var task = manager.GetSession(new Api.SerConnection()
+            {
+                ServerUri = new Uri("http://nb-fc-208000/ser"),
+                App = "dfacdb29-6cee-4cc6-b8b1-7a89014394dd",
+                Credentials = new Api.SerCredentials()
+                {
+                    Cert = certPath,
+                    PrivateKey = @"C:\Users\MBerthold\AppData\Roaming\senseexcel\reporting\serconnector_private.key",
```
Comment (konne) [inline src/Program.cs]: NO private paths inside the code

File: src/Communication.cs [hunk 2]
```diff
@@ -31,42 +38,40 @@ public class UserParameter
         public string BookmarkId { get; set; }
         public bool OnDemand { get; set; }
         public string PrivateKeyPath { get; set; }
+        public int CleanupTimeout { get; set; } = 20000;
```
Comment (konne) [inline src/Communication.cs]: do you handle that internal in miliseconds?

File: src/SerConAai.csproj [hunk 3]
```diff
@@ -79,11 +79,15 @@
     </PackageReference>
     <PackageReference Include="Hjson" Version="3.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="NLog" Version="5.0.0-beta11" />
+    <PackageReference Include="NLog" Version="4.5.4" />
```
Comment (konne) [inline src/SerConAai.csproj]: why you change back to 4.x.x ist this now also .netcore ready

**Comment by konne** [comment]
@mossbeard please fix the issue and answer the questions

File: src/SerConAai.csproj (hunk 3 above)
Comment (mossbeard) [reply src/SerConAai.csproj]: @konne 
The version 5.0.0 beta is no longer available. The max version is 4.5.4

File: src/Communication.cs (hunk 2 above)
Comment (mossbeard) [reply src/Communication.cs]: @konne 
Yes, i handle the time out in milliseconds.
If the property is set, I calculate x 1000

File: src/Program.cs (hunk 1 above)
Comment (mossbeard) [reply src/Program.cs]: @konne
It´s only for install qmc tests. It is not active.
I hand over the paths as parameters.

**Comment by mossbeard** [comment]
@konne comments are fixed.

**Pull request merged**